### PR TITLE
Add URL Shorter

### PR DESCRIPTION
**Pull request by jonasled**
added URL Shorter, written by myself

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x ] Your additions are ordered alphabetically.
- [x ] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [ x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [ x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x ] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x ] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x ] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [ x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x ] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1489,10 +1489,12 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [schort](https://github.com/sqozz/schort) - No login, no javascript, just short links. ([Demo](https://s.wx0.de)) `CC0-1.0` `Python`
 - [Shlink](https://shlink.io) - URL shortener with REST API and command line interface. Includes official progressive web application and docker images. ([Source Code](https://github.com/shlinkio/shlink), [Clients](https://shlink.io/apps)) `MIT` `PHP`
 - [shorturl](https://github.com/prologic/shorturl) - Simple URL shortener with very tiny URLs. ([Demo](https://url.mills.io)) `MIT` `Go`
+
```
Comment (SuperSandro2000) [inline README.md:1492]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1489,10 +1489,12 @@ See https://staticsitegenerators.net and https://www.staticgen.com
...
 - [Simple-URL-Shortener](https://github.com/azlux/Simple-URL-Shortener) - KISS URL shortener, public or private (with account). Minimalist and lightweight. No dependencies. ([Demo](https://u.azlux.fr)) `MIT` `PHP`
 - [url-shortener](https://github.com/cagataycali/url-shortener) `⚠` - Shitty url shortener, emoji and AI powered. `MIT` `Nodejs`
 - [URL-Shortener](https://github.com/sapioit/URL-shortener) - Make your long links short and only uses lowercase, so you can write and spell them faster. ([Demo](http://scurtez.cf)) `GPL-3.0` `PHP`
 - [YOURLS](http://yourls.org/) - YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener. Features include password protection, URL customization, bookmarklets, statistics, API, plugins, jsonp. ([Source Code](https://github.com/YOURLS/YOURLS)) `MIT` `PHP`
+[URL Shorter](https://gitlab.jonasled.de/jonasled/url_shorter_docker) - A easy to use URL shorter, written in Python with Flask and docker. [demo](https://kurz.ml)
```
Comment (SuperSandro2000) [inline README.md:1497]:
```suggestion
[URL Shorter](https://gitlab.jonasled.de/jonasled/url_shorter_docker) - A easy to use URL shorter, written in Python with Flask and docker. [demo](https://kurz.ml) `GPL-3.0` `Docker/Python`
```

**Comment by slel** [comment]
To make the issue title more telling, please rename this issue
from "Update README.md" to "Add URL Shorter".

The whole "awesome-selfhosted" project mostly consists in
a README.md file, so using "Update README.md" to name
a commit or an issue does not convey much about it.

**Comment by jonasled** [comment]
done

File: README.md [hunk 3]
```diff
@@ -1493,6 +1493,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [url-shortener](https://github.com/cagataycali/url-shortener) `⚠` - Shitty url shortener, emoji and AI powered. `MIT` `Nodejs`
 - [URL-Shortener](https://github.com/sapioit/URL-shortener) - Make your long links short and only uses lowercase, so you can write and spell them faster. ([Demo](http://scurtez.cf)) `GPL-3.0` `PHP`
 - [YOURLS](http://yourls.org/) - YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener. Features include password protection, URL customization, bookmarklets, statistics, API, plugins, jsonp. ([Source Code](https://github.com/YOURLS/YOURLS)) `MIT` `PHP`
+[URL Shorter](https://gitlab.jonasled.de/jonasled/url_shorter_docker) - A easy to use URL shorter, written in Python with Flask and docker. [demo](https://kurz.ml) `GPL-3.0` `Docker/Python`
```
Comment (n8225) [inline README.md:1496]: Change Docker/Python to just Python as this can be run without docker.

Comment (nodiscc) [reply README.md:1496]: If a Docker-based install is officially supported it does not harm to mention it? I think. However I think it should be the other way around (`Python/Docker`) as it is not *required*

**Comment by nodiscc** [comment]
No reply, closing

**Pull request closed without merging**